ert_gui/plottery/plot_style.py

```diff
@@ -65,10 +66,8 @@ def alpha(self):
 
     @alpha.setter
     def alpha(self, alpha):
-        if alpha > 1.0:
-            alpha = 1.0
-        if alpha < 0.0:
-            alpha = 0.0
+        alpha = min(alpha, 1.0)
+        alpha = max(alpha, 0.0)
```

Comment:
Very nitpick, but personally I think the clamp pattern can be written in one line like `alpha = max(min(alpha, 1.0), 0.0)`, but totally OK with leaving it as is